### PR TITLE
Merge reporting and reporter plugins

### PR DESCRIPTION
**Pull request by mlvandijk**

### CODE REVIEW

File: content/cucumber/reporting.md [hunk 1]
```diff
@@ -9,8 +11,18 @@ what scenarios have passed or failed.
...
+This page documents built-in formatter plugins as well common third-party plugins.
+Available plugins may vary per programming language.
+
 # Built-in reporter plugins
 
+There are several reporter plugins built into Cucumber:
+
+* progress
+* pretty
+* html
```
Comment (olleolleolle) [inline content/cucumber/reporting.md]: could these have teletype styling, like `progress`, so that they stand out next to the plaintext "etc."?

**Review by olleolleolle — APPROVED** [review]
This is great! Progress!

I made an inline note about styling, in order to keep it readable, even for new casual readers.

File: content/cucumber/reporting.md (hunk 1 above)
Comment (mlvandijk) [reply content/cucumber/reporting.md]: Sure, will do right away.

Comment (olleolleolle) [reply content/cucumber/reporting.md]: LGTM!

**Comment by mlvandijk** [comment]
Thnx for the quick review @olleolleolle

**Pull request merged**